L1Trigger/L1THGCal/python/hgcalTriggerPrimitiveDigiProducer_cfi.py

```diff
@@ -3,7 +3,13 @@
 fe_codec = cms.PSet( CodecName  = cms.string('HGCalBestChoiceCodec'),
                      CodecIndex = cms.uint32(1),
                      NData = cms.uint32(12),
-                     DataLength = cms.uint32(8)
+                     DataLength = cms.uint32(8),
+                     linLSB = cms.double(100./1024.),
+                     adcsaturation = cms.double(100),
```

Comment:
For parameters which are already defined in the HGC digitizer it is better to take them from there instead of redefining them. As it is done e.g. [here](https://github.com/PFCal-dev/cmssw/blob/hgc-tpg-devel-CMSSW_8_1_0_pre3/RecoLocalCalo%2FHGCalRecProducers%2Fpython%2FHGCalUncalibRecHit_cfi.py#L17-L24).